app/controllers/checkouts_controller.rb

```diff
@@ -138,7 +138,7 @@ def checkout_actions(checkout)
       elsif checkout.return_time < DateTime.current && checkout.media_object.lending_status == 'available'
         view_context.link_to('Checkout', checkouts_url(params: { checkout: { media_object_id: checkout.media_object_id } }), class: 'btn btn-primary btn-xs', method: :post)
```

Comment:
Should this case be removed as well to match the change in index.html.erb?

Reply:
Good question. @joncameron, If an item is not checked out should there be a checkout button next to the historical checkouts or should there only be `Return` buttons for current checkouts being rendered in the last column? 

It will probably be cleaner and more consistent to have all historical checkouts blank, and a user can just use the link to the item page to try to check something out again.

Reply:
@masaball I agree, it would be better to have all historical checkouts blank. Let's go that route.